comps/guardrails/factuality/predictionguard/factuality_predictionguard.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (C) 2024 Prediction Guard, Inc.
+# SPDX-License-Identified: Apache-2.0
+
+from predictionguard import PredictionGuard
+
+from comps import (
+    ServiceType, 
+    TextDoc, 
+    opea_microservices, 
+    register_microservice, 
+    register_statistics,
+    statistics_dict
+)
+
+
+@register_microservice(
+    name="opea_service@factuality_predictionguard",
+    service_type=ServiceType.GUARDRAIL,
+    endpoint="/v1/guardrails",
```

Comment:
I think we should give these endpoints a bit more descriptive names. Perhaps `/v1/factuality`